### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dishantsethi**

### CODE REVIEW

File: AbsoluteHack/AbsoluteHack/settings.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
```
Comment (AniVerma17) [inline AbsoluteHack/AbsoluteHack/settings.py:95]: Change the database to PostgreSQL.

**Review by AniVerma17 — CHANGES_REQUESTED** [review]

**Comment by AniVerma17** [comment]
@dishantsethi please give reference to the issue to be resolved by this PR in the description. Refer to the 'Pull Requests' section of [README](https://github.com/GlobalTechTalks/hackathon/blob/master/README.md) for referencing instructions.

File: AbsoluteHack/AbsoluteHack/settings.py (hunk 1 above)
Comment (dishantsethi) [reply AbsoluteHack/AbsoluteHack/settings.py:95]: okay

Comment (AniVerma17) [reply AbsoluteHack/AbsoluteHack/settings.py:95]: Alright, it's good now.

**Comment by AniVerma17** [comment]
> @dishantsethi please give reference to the issue to be resolved by this PR in the description. Refer to the 'Pull Requests' section of [README](https://github.com/GlobalTechTalks/hackathon/blob/master/README.md) for referencing instructions.

@dishantsethi please do this too (refer to the issue that was assigned to you) .
And squash your commits into one.

**Review by AniVerma17 — APPROVED** [review]

**Pull request closed without merging**